tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -72,6 +78,58 @@ def test_deploy_airgap_nodes():
         assert ag_node.public_ip_address is None
 
     deploy_airgap_nodes(bastion_node, ag_nodes)
+    results = add_cluster_to_rancher(bastion_node, ag_nodes)
+    for result in results:
+        assert "Downloaded newer image for {}/rancher/rancher-agent".format(
+            bastion_node.host_name) in result[1]
+
+
+def test_deploy_airgap_k3s():
+    bastion_node = get_bastion_node(BASTION_ID)
+
+    failures = add_k3s_images_to_private_registry(bastion_node,
+                                                  RANCHER_K3S_VERSION)
+    assert failures == [], "Failed to add images: {}".format(failures)
+    ag_nodes = prepare_airgap_k3s(bastion_node, NUMBER_OF_INSTANCES)
+    assert len(ag_nodes) == NUMBER_OF_INSTANCES
+
+    print(
+        '{} airgapped k3s instance(s) created.\n'
+        'Connect to these and run commands by connecting to bastion node, '
+        'then connecting to these:\n'
+        'ssh -i {}.pem {}@NODE_PRIVATE_IP'.format(
```

Comment:
Is  NODE_PRIVATE_IP is supposed to be a variable?

Reply:
No it's just supposed to print like that, so it shows up like:

```
Airgapped K3S Instance Details:
NAME: max-k3s-airgap-79619-0
PRIVATE IP: a.b.c.d

1 airgapped k3s instance(s) created.
Connect to these and run commands by connecting to bastion node, then connecting to these:
ssh -i thekey.pem ubuntu@NODE_PRIVATE_IP
```